### ISSUE TITLE
In the mobile version, there are problems in the Navigation drawer when scrolling.

### ISSUE THREAD

**Issue by cshishkov-7DIGIT** [UNICEFECAR/USupport-entry-bundle#55]
**Describe the bug**
In the mobile version, there are problems in the Navigation drawer when scrolling.

**To Reproduce**
Additional information is neede to reproduce the bug.

**Expected behavior**
No issue with the navigation when scrolling.

**Desktop:**
 - OS: [Mac]
 - Browser [Chrome, Safari, Firefox, Opera, Yandex]
 - Version [latest]

**Smartphone:**
 - Device: [iphone, android]
 - OS: [latest iso, latest android]
 - Browser [Chrome, Safari]
 - Version [latest]

**Additional context**
Additional information is needed to reproduce the bug.

**Comment by Gaskevych** [comment]
@gravitz maybe you can help with clarifications on this one?

**Comment by gravitz** [comment]
> @gravitz maybe you can help with clarifications on this one?

Is this about the bottom navigation (and not drawer, that was a separate suggestion that we have marked as needing further discussion) on the mobile app? If so, the current shading of the bottom navigation panel (tested on Android but the described behavior should be similar on iOS) is such that it is undifferentiable from the content panel that sits behind it. This can be observed as you scroll (on the home screen) and you happen to have an empty/blank area of the main screen (such as the suggestion text area input box) overlaid by the navigation panel. In this position, the panel (especially the appointment booking button) appears to be part of the component behind it. 

The proposed suggestion is to make the bottom nav bar more visually distinguishable from the rest of the screen.

**Comment by Gaskevych** [comment]
@sebastian-7DIGIT please refer to Engelbert's explanation above.

**Comment by sebastian-7DIGIT** [comment]
@Gaskevych @gravitz - It is clearer now what is the issue and the desired outcome. We identified the problem some time ago and implemented the adjustments. I will let you know once the changes are available to be visualised and tested on the mobile app as they will be released together with additional improvements soon.

**Comment by sebastian-7DIGIT** [comment]
The issue above was addressed under issue #65 and should be now available to be checked.

**Comment by mhdosman** [comment]
I think a touch (maybe a button like appearance) can make it more intutive, 
Device Samsung A53
![image](https://github.com/UNICEFECAR/USupport-entry-bundle/assets/153509442/158587fb-848f-480e-b786-ee4e49d1636a)
![WhatsApp Image 2024-02-09 at 14 50 41_b588ef5e](https://github.com/UNICEFECAR/USupport-entry-bundle/assets/153509442/1ffb5220-fd91-4663-bf81-1c513d8087c7)

**Comment by sebastian-7DIGIT** [comment]
@mhdosman noted! We will address this under issue #99 as is related to the same functionality.

**Comment by Gaskevych** [comment]
@sebastian-7DIGIT i dont think this issue has been addressed as it is still not displayed as a button but as a plain text

**Comment by sebastian-7DIGIT** [comment]
@Gaskevych , changes were deployed, ready to be tested.
